tests/console/login.go

```diff
@@ -116,11 +116,9 @@ func LoginToFedora(vmi *v1.VirtualMachineInstance) error {
 	// Do not login, if we already logged in
 	b := append([]expect.Batcher{
 		&expect.BSnd{S: "\n"},
-		&expect.BExp{R: PromptExpression},
-		&expect.BSnd{S: "echo $?\n"},
-		&expect.BExp{R: RetValue("0")},
+		&expect.BExp{R: fmt.Sprintf(`(\[fedora@(localhost|%s) ~\]\$ |\[root@(localhost|%s) fedora\]\# )`, vmi.Name, vmi.Name)},
```

Comment:
I don't fully understand the commit message. Can you please add an example of a failing scenario?

Reply:
Updated PR description, please let me know if its better now

Reply:
You wrote -
> Since the relogin tried to check that echo $? returns 0,
it would fail in the above case, since that last command returned non zero value.

But before the `echo $?` we have 
```
&expect.BSnd{S: "\n"},
&expect.BExp{R: PromptExpression},
```

That should clean the previous error from the buffer. So I still don't understand the scenario.

Reply:
sending \n doesnt clean error
```
~ $ false
~ $ 
~ $ echo $?
1
~ $ 
```

Reply:
Thanks for the explanation. Please add this example to the PR description.

Reply:
The fact that there is a send of `\n` just before it, doesn't consume the non zero return code.

Added this line